docs/basics/101-139-gin.rst

```diff
@@ -0,0 +1,214 @@
+.. _gin:
+
+Dataset hosting on GIN
+----------------------
+
+`GIN <https://gin.g-node.org/G-Node/Info/wiki>`__ (G-Node infrastructure) is a
+free data management system designed for comprehensive and reproducible management
+of scientific data. It is a secure, web-based repository store and provides
+fine-grained access control to share data. :term:`GIN` builds up on :term:`Git` and
+:term:`git-annex`, and is an easy alternative to other third-party services to host
+and share your DataLad datasets [#f1]_.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+In order to use GIN for hosting and sharing your datasets, you need to
+
+- register
+- upload your :term:`SSH key` for SSH access
+- create an empty repository on GIN and publish your dataset to it
+
+Once you have `registered <https://gin.g-node.org/user/sign_up>`_
+an account on the GIN server by providing your e-mail address, affiliation,
+and name, and selecting a user name and password, you should upload your
+:term:`SSH key` to allow SSH access.
+
+.. findoutmore:: What is an SSH key and how can I create one?
+
+   An SSH key is an access credential in the :term:`SSH` protocol that can be used
+   to login from one system to remote servers and services, such as from your private
+   computer to an :term:`SSH server`. For repository hosting services such as :term:`GIN`,
+   :term:`GitHub`, or :term:`GitLab`, it can be used to connect and authenticate
+   without supplying your username or password for each action.
+
+   This `tutorial by GitHub <https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent>`_
+   is a detailed step-by-step instruction to generate and use SSH keys for authentication,
+   and it also shows you how to add your public SSH key to your GitHub account
+   so that you can install or clone datasets or Git repositories via ``SSH`` (in addition
+   to the ``http`` protocol), and the same procedure applies to GitLab and Gin.
+
+   Don't be intimidated if you have never done this before -- it is fast and easy:
+   First, you need to create a private and a public key (an SSH key pair).
+   All this takes is a single command in the terminal. The resulting files are
+   text files that look like someone spilled alphabet soup in them, but constitute
+   a secure password procedure.
+   You keep the private key on your own machine (the system you are connecting from),
+   and copy the public key to the system or service you are connecting to.
+   On the remote system or service, you make the public key an *authorized key* to
+   allow authentication via the SSH key pair instead of your password. This
+   either takes a single command in the terminal, or a few clicks in a web interface
+   to achieve.
+   For extra security, you can protect your SSH keys on your machine with a
```

Comment:
I would recommend not calling it "extra security". Instead say something like "it should be protected by a passphrase to prevent..."